app/views/contacts/show.html.slim

```diff
@@ -0,0 +1,50 @@
+.row
+  .col-xs-12
+    h1= t_show
+
+.row
+  .col-xs-12
+    dl.dl-horizontal
```

Comment:
Why are you using dl, dt and dd tags? The other show views are solved with table